src/convertToObject.js

```diff
@@ -5,8 +5,48 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = function (sourceString) {
+  const styleObject = {};
+
+  const cleanSourceString = sourceString
+    .replace(/[\t\r]+/g, '')
+    .replace(/\s*;\s*/g, ';')
+    .replace(/\s*:\s*/g, ':')
+    .replace(/;\n/g, ';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy of the source string and perform multiple replace operations. You can directly split and process the string in one go using iteration methods.